src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php

```diff
@@ -60,6 +60,10 @@ public function generateAbsoluteUrl($path)
             return $path;
         }
 
+        if ('/' === $path[0]) {
```

Comment:
If `$path` is empty we want to keep the base path no ? I would expect `absolute_url('')` to return `http://localhost/app/web/` instead of `http://localhost`